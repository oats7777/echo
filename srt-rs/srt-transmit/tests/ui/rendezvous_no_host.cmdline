["udp://:4002" ,  "srt://:4003?rendezvous"]