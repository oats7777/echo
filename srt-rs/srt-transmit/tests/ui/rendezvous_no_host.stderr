Invalid settings detected: Cannot have rendezvous connection without host specified

See srt-transmit --help for more info
